src/lib/routing-table.js

```diff
@@ -70,29 +70,31 @@ class RoutingTable {
       return undefined
     }
 
-    let bestHop = null
-    let bestValue = null
-    let bestRoute = null
+    let bestHop
     routes.forEach((route, nextHop) => {
-      const value = route.amountAt(sourceAmount)
       // If we have a route but not a curve, pick a route at random
       // and get a remote quote. In the future we may refactor this
       // so that multiple next hop options can be returned, and all
       // can be asked for a quote, but for now, it's just the last.
-      if (value === undefined || !bestValue || value.gt(bestValue)) {
-        bestHop = nextHop
-        bestValue = value && value.toString()
-        bestRoute = route
-      }
+      bestHop = getBetterHop(bestHop, {
+        nextHop,
+        route,
+        value: route.amountAt(sourceAmount),
+        pathLength: route.maxPathLength()
+      })
     })
 
     if (bestHop) {
-      debug('findBestHopForSourceAmount to ' + destination + ' for ' + sourceAmount + ' found route through ' + bestHop)
+      debug('findBestHopForSourceAmount to ' + destination + ' for ' + sourceAmount + ' found route through ' + bestHop.nextHop)
```

Comment:
See above, `bestHop.nextHop` - I don't know what that means, the next Hop of the best Hop. Which one is the Route object here and which one is the peer ledger name String?